en/docs/setup/deployment-guide.md

```diff
@@ -277,13 +276,13 @@ WSO2 supports the following membership schemes for clustering
                     vpcCidrBlock = "10.0.*.*"
                     ```                    
             Under the `clustering.properties` section, set the `region`, `clusterName`, and `vpcCidrBlock` based on 
-            the AWS ECS cluster that we crated in the previous step.       
+            the AWS ECS cluster that you crated in the previous step.       
```

Comment:
Small typo - "crated" should be corrected to "created"

Reply:
Resolved in https://github.com/wso2/docs-is/pull/895/commits/2fe199e6759c1b50721820f1f54cd60f0c2911b4